### PR TITLE
[FLINK-30838][doc] Update documentation about the AdaptiveBatchScheduler

### PR DESCRIPTION
**Pull request by JunRuiLee**
## What is the purpose of the change

Update documentation about the AdaptiveBatchScheduler

## Verifying this change
Document change without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

**Comment by JunRuiLee** [comment]
@wanglijie95 Thanks for CR. I've addressed all comments. Please take a look.

File: docs/content.zh/docs/deployment/elastic_scaling.md [hunk 1]
```diff
@@ -159,30 +159,33 @@ Adaptive Batch Scheduler 是一种可以自动推导每个算子并行度的批
 
 使用 Adaptive Batch Scheduler 自动推导算子的并行度，需要：
 - 启用 Adaptive Batch Scheduler
-- 配置算子的并行度为 `-1`
+- 不要明确指定算子的并发度
 
 #### 启用 Adaptive Batch Scheduler
-为了启用 Adaptive Batch Scheduler, 你需要：
-- 配置 `jobmanager.scheduler: AdaptiveBatch`
+当前Adaptive Batch Scheduler是flink默认的批调度器，为了启用 Adaptive Batch Scheduler, 你需要：
```
Comment (zhuzhurk) [inline docs/content.zh/docs/deployment/elastic_scaling.md:165]:
```suggestion
当前 Adaptive Batch Scheduler 是 Flink 默认的批作业调度器，无需额外配置。除非用户显式的配置了使用其他调度器，例如 `jobmanager.scheduler: default`。需要注意的是，由于...
```
It's better to add a space between en&zh characters.

File: docs/content.zh/docs/deployment/elastic_scaling.md [hunk 2]
```diff
@@ -159,30 +159,33 @@ Adaptive Batch Scheduler 是一种可以自动推导每个算子并行度的批
...
+- [`execution.batch.adaptive.auto-parallelism.min-parallelism`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-min-parallelism): 允许自动设置的并行度最小值。
+- [`execution.batch.adaptive.auto-parallelism.max-parallelism`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-max-parallelism): 允许自动设置的并行度最大值，如果该配置项没有配置将使用默认并发度作为允许自动设置的并发度最大值。
+- [`execution.batch.adaptive.auto-parallelism.avg-data-volume-per-task`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-avg-data-volume-per-task): 期望每个任务平均处理的数据量大小。请注意，当出现数据倾斜，或者确定的并行度达到最大并行度（由于数据过多）时，一些任务实际处理的数据可能会远远超过这个值。
+- [`execution.batch.adaptive.auto-parallelism.default-source-parallelism`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-default-source-parallelism): source 算子的默认并行度
+
+#### 不要明确指定算子的并发度
+Adaptive Batch Scheduler 只会为用户未指定并行度的算子（DataSet作业并行度为 `-1`，DataStream作业并行度为 `-1`或者默认并行度）推导并行度。 所以如果你想自动推导算子的并行度，对于DataStream作业需要进行以下配置：
+- 对于 DataStream 作业，不要在作业中通过算子的 `setParallelism()` 方法来指定并行度
+- 对于 DataStream 作业，不要在作业中通过 `StreamExecutionEnvironment`  的 `setParallelism()` 方法来指定并行度
```
Comment (zhuzhurk) [inline docs/content.zh/docs/deployment/elastic_scaling.md:177]: The statement is incorrect and should be removed.

File: docs/content.zh/docs/deployment/elastic_scaling.md [hunk 3]
```diff
@@ -159,30 +159,33 @@ Adaptive Batch Scheduler 是一种可以自动推导每个算子并行度的批
...
-
-#### 配置算子的并行度为 `-1`
-Adaptive Batch Scheduler 只会为用户未指定并行度的算子（并行度为 `-1`）推导并行度。 所以如果你想自动推导算子的并行度，需要进行以下配置：
+- [`execution.batch.adaptive.auto-parallelism.min-parallelism`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-min-parallelism): 允许自动设置的并行度最小值。
+- [`execution.batch.adaptive.auto-parallelism.max-parallelism`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-max-parallelism): 允许自动设置的并行度最大值，如果该配置项没有配置将使用默认并发度作为允许自动设置的并发度最大值。
+- [`execution.batch.adaptive.auto-parallelism.avg-data-volume-per-task`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-avg-data-volume-per-task): 期望每个任务平均处理的数据量大小。请注意，当出现数据倾斜，或者确定的并行度达到最大并行度（由于数据过多）时，一些任务实际处理的数据可能会远远超过这个值。
+- [`execution.batch.adaptive.auto-parallelism.default-source-parallelism`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-default-source-parallelism): source 算子的默认并行度
+
+#### 不要明确指定算子的并发度
+Adaptive Batch Scheduler 只会为用户未指定并行度的算子（DataSet作业并行度为 `-1`，DataStream作业并行度为 `-1`或者默认并行度）推导并行度。 所以如果你想自动推导算子的并行度，对于DataStream作业需要进行以下配置：
```
Comment (zhuzhurk) [inline docs/content.zh/docs/deployment/elastic_scaling.md:175]:
```suggestion
Adaptive Batch Scheduler 只会为用户未指定并行度的算子推导并行度。 所以如果你想算子的并行度被自动推导，需要避免通过算子的 `setParallelism()` 方法来为其指定并行度。
```

File: docs/content.zh/docs/deployment/elastic_scaling.md [hunk 4]
```diff
@@ -159,30 +159,33 @@ Adaptive Batch Scheduler 是一种可以自动推导每个算子并行度的批
...
+- 对于 DataStream 作业，不要在作业中通过算子的 `setParallelism()` 方法来指定并行度
+- 对于 DataStream 作业，不要在作业中通过 `StreamExecutionEnvironment`  的 `setParallelism()` 方法来指定并行度
+####
```
Comment (zhuzhurk) [inline docs/content.zh/docs/deployment/elastic_scaling.md:178]: This empty subtitle should be removed.

File: docs/content.zh/docs/deployment/elastic_scaling.md [hunk 5]
```diff
@@ -159,30 +159,33 @@ Adaptive Batch Scheduler 是一种可以自动推导每个算子并行度的批
...
+对于DataSet作业需要进行以下配置：
```
Comment (zhuzhurk) [inline docs/content.zh/docs/deployment/elastic_scaling.md:179]:
```suggestion
除此之外，对于 DataSet 作业还需要进行以下配置：
```

File: docs/content.zh/docs/deployment/elastic_scaling.md [hunk 6]
```diff
@@ -159,30 +159,33 @@ Adaptive Batch Scheduler 是一种可以自动推导每个算子并行度的批
...
 - 配置 `parallelism.default: -1`
 - 对于 SQL 作业，需要配置 `table.exec.resource.default-parallelism: -1`
```
Comment (zhuzhurk) [inline docs/content.zh/docs/deployment/elastic_scaling.md:181]: This line should be removed.

File: docs/content.zh/docs/deployment/elastic_scaling.md [hunk 7]
```diff
@@ -159,30 +159,33 @@ Adaptive Batch Scheduler 是一种可以自动推导每个算子并行度的批
...
-- 对于 DataStream/DataSet 作业，不要在作业中通过算子的 `setParallelism()` 方法来指定并行度
-- 对于 DataStream/DataSet 作业，不要在作业中通过 `StreamExecutionEnvironment/ExecutionEnvironment` 的 `setParallelism()` 方法来指定并行度
+- 对于 DataSet 作业，不要在作业中通过算子的 `setParallelism()` 方法来指定并行度
```
Comment (zhuzhurk) [inline docs/content.zh/docs/deployment/elastic_scaling.md:182]: This line can be removed.

File: docs/content.zh/docs/deployment/elastic_scaling.md [hunk 8]
```diff
@@ -159,30 +159,33 @@ Adaptive Batch Scheduler 是一种可以自动推导每个算子并行度的批
...
+- 对于 DataSet 作业，不要在作业中通过 `ExecutionEnvironment` 的 `setParallelism()` 方法来指定并行度
```
Comment (zhuzhurk) [inline docs/content.zh/docs/deployment/elastic_scaling.md:183]:
```suggestion
- 不要通过 `ExecutionEnvironment` 的 `setParallelism()` 方法来指定并行度
```

**Review by zhuzhurk — COMMENTED** [review]
Thanks for creating this PR.  @JunRuiLee 
I have a few comments. PTAL

**Comment by JunRuiLee** [comment]
@zhuzhurk Thanks for CR and I've addressed all comments, PTAL.

File: docs/content/docs/deployment/elastic_scaling.md [hunk 9]
```diff
@@ -161,30 +161,28 @@ The Adaptive Batch Scheduler can automatically decide parallelisms of operators
...
-
-#### Set the parallelism of operators to `-1`
-Adaptive Batch Scheduler will only decide parallelism for operators whose parallelism is not specified by users (parallelism is `-1`). So if you want the parallelism of operators to be decided automatically, you should configure as follows:
-- Set `parallelism.default: -1`
-- Set `table.exec.resource.default-parallelism: -1` in SQL jobs.
-- Don't call `setParallelism()` for operators in DataStream/DataSet jobs.
-- Don't call `setParallelism()` on `StreamExecutionEnvironment/ExecutionEnvironment` in DataStream/DataSet jobs.
+- [`execution.batch.adaptive.auto-parallelism.min-parallelism`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-min-parallelism): The lower bound of allowed parallelism to set adaptively.
+- [`execution.batch.adaptive.auto-parallelism.max-parallelism`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-max-parallelism): The upper bound of allowed parallelism to set adaptively. Default parallelism will be used as upper bound of allowed parallelism if this configuration is not configured.
```
Comment (zhuzhurk) [inline docs/content/docs/deployment/elastic_scaling.md:172]: Default parallelism -> The default parallelism set via [\`parallelism.default\`]({{< ref "docs/deployment/config" >}}) or \`StreamExecutionEnvironment#setParallelism()\`

File: docs/content/docs/deployment/elastic_scaling.md [hunk 10]
```diff
@@ -161,30 +161,28 @@ The Adaptive Batch Scheduler can automatically decide parallelisms of operators
...
+- [`execution.batch.adaptive.auto-parallelism.avg-data-volume-per-task`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-avg-data-volume-per-task): The average size of data volume to expect each task instance to process. Note that when data skew occurs, or the decided parallelism reaches the max parallelism (due to too much data), the data actually processed by some tasks may far exceed this value.
+- [`execution.batch.adaptive.auto-parallelism.default-source-parallelism`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-default-source-parallelism): The default parallelism of data source.
+
+#### Avoid specify the parallelism of operators
```
Comment (zhuzhurk) [inline docs/content/docs/deployment/elastic_scaling.md:176]: specify -> setting

File: docs/content/docs/deployment/elastic_scaling.md [hunk 11]
```diff
@@ -161,30 +161,28 @@ The Adaptive Batch Scheduler can automatically decide parallelisms of operators
...
+The Adaptive Batch Scheduler only decides the parallelism for operators for which the user has not specified the parallelism. So if you want the parallelism of the operator to be automatically decided, you need to avoid specifying the parallelism for the operator through the 'setParallelism()' method.
```
Comment (zhuzhurk) [inline docs/content/docs/deployment/elastic_scaling.md:177]: for which the user has not specified the parallelism -> which do not have a parallelism set
the operator -> an operator
avoid specifying -> avoid setting

File: docs/content/docs/deployment/elastic_scaling.md [hunk 12]
```diff
@@ -161,30 +161,28 @@ The Adaptive Batch Scheduler can automatically decide parallelisms of operators
...
+In addition, the following configurations are required for DataSet jobs:
+- Set `parallelism.default: -1`.
+- Don't call `setParallelism()` on `ExecutionEnvironment` in DataSet jobs.
```
Comment (zhuzhurk) [inline docs/content/docs/deployment/elastic_scaling.md:180]: remove "in DataSet jobs".

File: docs/content.zh/docs/deployment/speculative_execution.md [hunk 13]
```diff
@@ -45,15 +45,15 @@ under the License.
 {{< /hint >}}
 
 ### 启用预测执行
-要启用预测执行，你需要设置以下配置项：
+要启用预测执行，你需要不显式设置配置项`jobmanager.scheduler`或者按如下方式配置：
 - `jobmanager.scheduler: AdaptiveBatch`
     - 因为当前只有 [Adaptive Batch Scheduler]({{< ref "docs/deployment/elastic_scaling" >}}#adaptive-batch-scheduler) 支持预测执行.
-- `jobmanager.adaptive-batch-scheduler.speculative.enabled: true`
+- `execution.batch.speculative.enabled: true`
```
Comment (zhuzhurk) [inline docs/content.zh/docs/deployment/speculative_execution.md:51]: I would rework the above part as below:
```
你可以通过以下配置项启用预测执行：
- `execution.batch.speculative.enabled: true`

需要注意的是，当前只有 [Adaptive Batch Scheduler]({{< ref "docs/deployment/elastic_scaling" >}}#adaptive-batch-scheduler) 支持预测执行。不过 Flink 批作业会默认使用该调度器，除非显式配置了其他调度器。
```

**Comment by JunRuiLee** [comment]
@zhuzhurk Thanks for CR and I've addressed all comments, PTAL.

File: docs/content/docs/deployment/elastic_scaling.md (hunk 9 above)
Comment (zhuzhurk) [reply docs/content/docs/deployment/elastic_scaling.md]: The zh doc should be updated accordingly.

File: docs/content.zh/docs/deployment/speculative_execution.md [hunk 14]
```diff
@@ -45,15 +45,15 @@ under the License.
 {{< /hint >}}
 
 ### 启用预测执行
-要启用预测执行，你需要设置以下配置项：
-- `jobmanager.scheduler: AdaptiveBatch`
-    - 因为当前只有 [Adaptive Batch Scheduler]({{< ref "docs/deployment/elastic_scaling" >}}#adaptive-batch-scheduler) 支持预测执行.
-- `jobmanager.adaptive-batch-scheduler.speculative.enabled: true`
+你可以通过以下配置项启用预测执行：
+- `execution.batch.speculative.enabled: true`
+####
```
Comment (zhuzhurk) [inline docs/content.zh/docs/deployment/speculative_execution.md:50]: This line should be replaced with an empty new line.

File: docs/content/docs/deployment/speculative_execution.md [hunk 15]
```diff
@@ -52,15 +52,14 @@ in near future. DataStream API is now the recommended low level API to develop F
...
-To enable speculative execution, you need to set the following configuration options:
-- `jobmanager.scheduler: AdaptiveBatch`
-    - Because only [Adaptive Batch Scheduler]({{< ref "docs/deployment/elastic_scaling" >}}#adaptive-batch-scheduler) supports speculative execution.
-- `jobmanager.adaptive-batch-scheduler.speculative.enabled: true`
-
+You can enable speculative execution through the following configuration items：
+- `execution.batch.speculative.enabled: true`
+####
+Note that currently only [Adaptive Batch Scheduler]({{< ref "docs/deployment/elastic_scaling" >}}#adaptive-batch-scheduler) supports speculative execution. And Flink batch jobs will use this scheduler by default unless another scheduler is explicitly configured.
 ### Tuning Configuration
```
Comment (zhuzhurk) [inline docs/content/docs/deployment/speculative_execution.md:59]: An empty line should be added in above.

**Review by zhuzhurk — APPROVED** [review]
Thanks for addressing the comments. @JunRuiLee 
LGTM.

**Comment by zhuzhurk** [comment]
Well, I just noticed we mixed the concept of AdaptiveBatchScheduler and auto-parallelism. They were almost one thing in the past but are different now. We should re-organize the doc.

**Review by zhuzhurk — CHANGES_REQUESTED** [review]
Well, I just noticed we mixed the concept of AdaptiveBatchScheduler and auto-parallelism. They were almost one thing in the past but are different now. We should re-organize the doc.

File: docs/content.zh/docs/deployment/elastic_scaling.md [hunk 16]
```diff
@@ -161,14 +161,15 @@ Adaptive Batch Scheduler 是一种可以自动推导每个算子并行度的批
 - 启用 Adaptive Batch Scheduler
 - 不要指定算子的并行度
 
-#### 启用 Adaptive Batch Scheduler
+#### 启用 Adaptive Batch Scheduler 自动推导并行度
 当前 Adaptive Batch Scheduler 是 Flink 默认的批作业调度器，无需额外配置。除非用户显式的配置了使用其他调度器，例如 `jobmanager.scheduler: default`。需要注意的是，由于 ["只支持所有数据交换都为 BLOCKING 模式的作业"](#局限性-2), 需要将 [`execution.batch-shuffle-mode`]({{< ref "docs/deployment/config" >}}#execution-batch-shuffle-mode) 配置为 `ALL-EXCHANGES-BLOCKING`(默认值) 。
 
-除此之外，使用 Adaptive Batch Scheduler 时，以下相关配置也可以调整:
+除此之外，使用 Adaptive Batch Scheduler 自动推导并行度时，以下相关配置也可以调整:
+- [`execution.batch.adaptive.auto-parallelism.enabled`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-enabled): 是否开启并行度推导，默认开启。
```
Comment (zhuzhurk) [inline docs/content.zh/docs/deployment/elastic_scaling.md:168]: I would refrase it as "Adaptive Batch Scheduler 默认启用了自动并行度推导，你可以通过 xxx 配置来开关此功能。除此之外，你也可以根据作业的情况调整以下配置:"

File: docs/content/docs/deployment/elastic_scaling.md [hunk 17]
```diff
@@ -152,7 +152,7 @@ The behavior of Adaptive Scheduler is configured by [all configuration options c
 
 ## Adaptive Batch Scheduler
 
-The Adaptive Batch Scheduler can automatically decide parallelisms of operators for batch jobs. If an operator is not set with a parallelism, the scheduler will decide parallelism for it according to the size of its consumed datasets. This can bring many benefits:
+The Adaptive Batch Scheduler is a batch job scheduler that can automatically adjust the execution plan and currently support automatically decide parallelisms of operators for batch jobs. If an operator is not set with a parallelism, the scheduler will decide parallelism for it according to the size of its consumed datasets. This can bring many benefits:
```
Comment (zhuzhurk) [inline docs/content/docs/deployment/elastic_scaling.md:155]: plan and currently -> plan. It currently
support -> supports
decide -> deciding

File: docs/content/docs/deployment/elastic_scaling.md [hunk 18]
```diff
@@ -190,6 +191,6 @@ In addition, the following configurations are required for DataSet jobs:
...
-- **Inconsistent broadcast results metrics on WebUI**: In Adaptive Batch Scheduler, for broadcast results, the number of bytes/records sent by the upstream task counted by metric is not equal to the number of bytes/records received by the downstream task, which may confuse users when displayed on the Web UI. See [FLIP-187](https://cwiki.apache.org/confluence/display/FLINK/FLIP-187%3A+Adaptive+Batch+Job+Scheduler) for details.
+- **Inconsistent broadcast results metrics on WebUI**: When use Adaptive Batch Scheduler automatically decide parallelisms for operators, for broadcast results, the number of bytes/records sent by the upstream task counted by metric is not equal to the number of bytes/records received by the downstream task, which may confuse users when displayed on the Web UI. See [FLIP-187](https://cwiki.apache.org/confluence/display/FLINK/FLIP-187%3A+Adaptive+Batch+Job+Scheduler) for details.
```
Comment (zhuzhurk) [inline docs/content/docs/deployment/elastic_scaling.md:194]: automatically -> to automatically

File: docs/content/docs/deployment/elastic_scaling.md [hunk 19]
```diff
@@ -161,36 +161,37 @@ The Adaptive Batch Scheduler can automatically decide parallelisms of operators
...
-- Set `parallelism.default: -1`
-- Set `table.exec.resource.default-parallelism: -1` in SQL jobs.
-- Don't call `setParallelism()` for operators in DataStream/DataSet jobs.
-- Don't call `setParallelism()` on `StreamExecutionEnvironment/ExecutionEnvironment` in DataStream/DataSet jobs.
+At present, the Adaptive Batch Scheduler is the default scheduler for Flink batch jobs. No additional configuration is required unless other schedulers are explicitly configured, e.g. `jobmanager.scheduler: default`. Note that you need to 
+leave the [`execution.batch-shuffle-mode`]({{< ref "docs/deployment/config" >}}#execution-batch-shuffle-mode) unset or explicitly set it to `ALL-EXCHANGES-BLOCKING` (default value) due to ["ALL-EXCHANGES-BLOCKING jobs only"](#limitations-2).
+
+#### Configure to automatically decide parallelisms for operators
+Adaptive Batch Scheduler enables automatic parallelism derivation by default, you can configure [`execution.batch.adaptive.auto-parallelism.enabled`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-enabled) to switch this feature.
```
Comment (zhuzhurk) [inline docs/content/docs/deployment/elastic_scaling.md:171]: switch -> toggle

File: docs/content/docs/deployment/elastic_scaling.md [hunk 20]
```diff
@@ -161,36 +161,37 @@ The Adaptive Batch Scheduler can automatically decide parallelisms of operators
...
+In addition, there are several related configuration options that may need adjustment when using Adaptive Batch Scheduler automatically decide parallelisms for operators:
```
Comment (zhuzhurk) [inline docs/content/docs/deployment/elastic_scaling.md:172]: automatically -> to automatically

File: docs/content/docs/deployment/elastic_scaling.md [hunk 21]
```diff
@@ -152,45 +152,46 @@ The behavior of Adaptive Scheduler is configured by [all configuration options c
...
+leave the [`execution.batch-shuffle-mode`]({{< ref "docs/deployment/config" >}}#execution-batch-shuffle-mode) unset or explicitly set it to `ALL-EXCHANGES-BLOCKING` (default value) due to ["ALL-EXCHANGES-BLOCKING jobs only"](#limitations-2).
+
+### Automatically decide parallelisms for operators
+
+#### Usage
 
 To automatically decide parallelisms for operators with Adaptive Batch Scheduler, you need to:
-- Configure to use Adaptive Batch Scheduler.
-- Set the parallelism of operators to `-1`.
+- Configure to automatically decide parallelisms for operators:
```
Comment (zhuzhurk) [inline docs/content/docs/deployment/elastic_scaling.md:168]:
```suggestion
- Toggle the feature on:
```

File: docs/content/docs/deployment/elastic_scaling.md [hunk 22]
```diff
@@ -152,45 +152,46 @@ The behavior of Adaptive Scheduler is configured by [all configuration options c
...
-
-#### Set the parallelism of operators to `-1`
-Adaptive Batch Scheduler will only decide parallelism for operators whose parallelism is not specified by users (parallelism is `-1`). So if you want the parallelism of operators to be decided automatically, you should configure as follows:
-- Set `parallelism.default: -1`
-- Set `table.exec.resource.default-parallelism: -1` in SQL jobs.
-- Don't call `setParallelism()` for operators in DataStream/DataSet jobs.
-- Don't call `setParallelism()` on `StreamExecutionEnvironment/ExecutionEnvironment` in DataStream/DataSet jobs.
-
-### Performance tuning
+    Adaptive Batch Scheduler enables automatic parallelism derivation by default, you can configure [`execution.batch.adaptive.auto-parallelism.enabled`]({{< ref "docs/deployment/config" >}}#execution-batch-adaptive-auto-parallelism-enabled) to toggle this feature.
```
Comment (zhuzhurk) [inline docs/content/docs/deployment/elastic_scaling.md:170]: , you -> . You

**Review by zhuzhurk — APPROVED** [review]
LGTM.

**Comment by zhuzhurk** [comment]
@flinkbot run azure

**Pull request closed without merging**